backend/src/main/scala/sbt/internal/inc/BloopComponentCompiler.scala

```diff
@@ -386,8 +380,8 @@ private[inc] class BloopComponentCompiler(
           val allSourceContents =
             (hydraSourceContents ++ regularSourceContents).map(s => s -> relativize(tempDir, s).get)
 
-          zip(allSourceContents.toSeq, mergedJar)
-          Right(Vector(mergedJar))
+          zip(allSourceContents.toSeq, mergedJar, time = None)
```

Comment:
:+1: Do we have other palces in bloop where we zip some files?

Reply:
Not from what I can see, this is for the compiler bridge compiled sources.